upb/def.c

```diff
@@ -1654,6 +1655,19 @@ static void make_layout(symtab_addctx *ctx, const upb_msgdef *m) {
   /* Assign hasbits for required fields first. */
   size_t hasbit = 0;
 
+  for (int i = 0; i < m->field_count; i++) {
+    const upb_fielddef* f = &m->fields[i];
+    upb_msglayout_field *field = &fields[upb_fielddef_index(f)];
+    if (upb_fielddef_label(f) == UPB_LABEL_REQUIRED) {
+      field->presence = ++hasbit;
+      if (hasbit >= 63) {
+        symtab_errf(ctx, "Message with >=63 required fields: %s",
```

Comment:
This is `longjmp()` again, this is aborting the whole operation.

I tried doing what we did with `longjmp()` in the decoder, by changing this to `return symtab_errf()`. But it didn't work here, because the function returns `void` (unlike C++, C won't let you `return f();` in a function returning void).